suite-native/atoms/src/AlertBox.tsx

```diff
@@ -1,46 +1,92 @@
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
+import { Color } from '@trezor/theme';
+import { Icon, IconName } from '@suite-common/icons';
 
 import { Box } from './Box';
 import { Text } from './Text';
-import { RoundedIcon } from './RoundedIcon';
+
+type AlertBoxVariant = 'info' | 'success' | 'warning' | 'error';
+
+type AlertBoxStyle = {
+    backgroundColor: Color;
+    contentColor: Color;
+};
 
 type AlertBoxProps = {
-    title: string;
-    isIconVisible?: boolean;
+    title?: React.ReactNode;
+    isStandalone?: boolean;
+    variant: AlertBoxVariant;
 };
 
-const alertWrapperStyle = prepareNativeStyle(utils => ({
-    flexDirection: 'row',
-    alignItems: 'center',
-    padding: utils.spacings.medium,
-    borderRadius: utils.borders.radii.medium,
-    backgroundColor: utils.colors.backgroundAlertBlueSubtleOnElevation0,
-}));
+type AlertWrapperStyleType = {
+    isStandalone: boolean;
+    backgroundColor: Color;
+};
+
+const alertWrapperStyle = prepareNativeStyle<AlertWrapperStyleType>(
+    (utils, { isStandalone, backgroundColor }) => ({
+        flexDirection: 'row',
+        alignItems: 'center',
+        borderRadius: isStandalone ? utils.borders.radii.small : 0,
+        backgroundColor: utils.colors[backgroundColor],
+        paddingVertical: utils.spacings.small,
+        paddingHorizontal: utils.spacings.medium,
+        gap: utils.spacings.small,
+    }),
+);
 
-const textWidthStyle = prepareNativeStyle(_ => ({
+const textStyle = prepareNativeStyle(_ => ({
     flex: 1,
 }));
 
-export const AlertBox = ({ title, isIconVisible = true }: AlertBoxProps) => {
+const variantToColorMap = {
+    info: {
+        backgroundColor: 'backgroundAlertBlueSubtleOnElevation0',
+        contentColor: 'iconAlertBlue',
+    },
+    success: {
+        backgroundColor: 'backgroundPrimarySubtleOnElevation0',
+        contentColor: 'textSecondaryHighlight',
+    },
+    warning: {
+        backgroundColor: 'backgroundAlertYellowSubtleOnElevation0',
+        contentColor: 'textAlertYellow',
+    },
+    error: {
+        backgroundColor: 'backgroundAlertRedSubtleOnElevation0',
+        contentColor: 'textAlertRed',
+    },
+} as const satisfies Record<AlertBoxVariant, AlertBoxStyle>;
+
+const variantToIconName = {
+    info: 'info',
+    success: 'checkCircle',
+    warning: 'warningTriangle',
+    error: 'warningTriangle',
+} as const satisfies Record<AlertBoxVariant, IconName>;
+
+export const AlertBox = ({ title, isStandalone = false, variant = 'info' }: AlertBoxProps) => {
     const { applyStyle } = useNativeStyles();
 
```

Comment:
nit:
```suggestion
   const {contentColor, backgroundColor} = variantToColorMap[variant];
```

Like this, you don't have to use the long version (e.g. `variantToColorMap[variant].contentColor`) multiple times.
But this is minor thing. If you don't wanna change this, I'm OK with that.

Reply:
nice, updated in 9b4dd74f2